allowed-terraform-version.sentinel

```diff
@@ -1,5 +1,7 @@
 import "tfplan"
 
+# This regular expression checks whether the Terraform version used for the plan is 0.14+, 0.15+, or 1.0+
+
 main = rule {
-  tfplan.terraform_version matches "^(0\\.1(4|5)|1\\.0)\\.\\d+$"
+  tfplan.terraform_version matches "^(0\\.1(4|5)|1\\.\\d)\\.\\d+$"
```

Comment:
I don't think this works for two digit minor versions, like `1.11.0`. I think we want `"^(0\\.1(4|5)|1\\.\\d+)\\.\\d+$"` instead.